src/runtime/vm/vm.cc

```diff
@@ -221,6 +211,9 @@ PackedFunc VirtualMachine::GetFunction(const std::string& name,
   } else if (name == "set_input") {
     return PackedFunc(
         [sptr_to_self, this](TVMArgs args, TVMRetValue* rv) { SetInput(args[0], args, 1); });
+  } else if (name == "set_one_input") {
```

Comment:
nit: suggest a unit test  exercising this new method from vm.py (though I'm assuming you need to change that anyway, perhaps in a follow up PR?) Maybe you are just using the runtime module directly?

Reply:
Yes, I use runtime module directly on native side. It is a reason why I try to use `set_input` methods instead of direct using of `invoke`. The latter requires to pack all input tensors to TvmArgs in certain sequence on native side. For me it is more controlable and clear to input one by one tensor with its tag (index or name). It is simultaneously the answer to your below question. Nevertheless I agree that my patch was not full and I've extended python API of VirtualMachine by `set_one_input` method. I observed that there is no even `set_input` unit test. Unit tests are in progress.

Reply:
I've added pytests.